lib/bolt/pal/yaml_plan.rb

```diff
@@ -1,10 +1,14 @@
 # frozen_string_literal: true
 
+require 'puppet_pal'
+require 'bolt/pal/yaml_plan/parameter'
+require 'bolt/pal/yaml_plan/step'
+
 module Bolt
   class PAL
     class YamlPlan
       PLAN_KEYS = Set['parameters', 'steps', 'return', 'version']
-      PARAMETER_KEYS = Set['type', 'default']
+      PARAMETER_KEYS = Set['type', 'default', 'description']
```

Comment:
I did not realize this was an allowable key. I put up a separate PR https://github.com/puppetlabs/bolt/pull/968 to get in to release without being blocked on this PR.